rfc/0054-sdk3-management-apis.md

```diff
@@ -342,12 +342,29 @@ Iterable<QueryIndex> GetAllIndexes(string bucketName, [options])
 * Optional:
 
   * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+  * `CollectionName` - the name of the collection to restrict indexes to.
+  * `ScopeName` - the name of the scope to restrict indexes to.
```

Comment:
NIT: Is it worth mentioning that if `CollectionName` is specified, `ScopeName` must also be specified?